Detector/DetFCChhECalInclined/compact/FCChh_ECalBarrel_Common.xml

```diff
@@ -60,7 +60,7 @@
     <!-- phi position is calculated based on the centre of volume (hence it cannot be done in the simulation from energy deposits position) -->
     <readout name="ECalBarrelPhiEta">
       <segmentation type="GridPhiEta" grid_size_eta="0.01" phi_bins="704" offset_eta="-1.68024" offset_phi="-pi+(pi/704.)"/>
-      <id>system:4,cryo:1,module:11,type:3,subtype:3,layer:8,eta:9,phi:10</id>
```

Comment:
Hi Jana, 
I merged with your branch and get some issues with the matching between CellID and volumeID (for the retrieval of cell positions) because of the removed "module" field in the ECAL readout. Could we keep this in, or do you have a strong reason for rm it?